modules.js

```diff
@@ -1,9 +1,9 @@
-var debug = require('debug')('objectify-folder');
-var fs = require('fs');
-var glob = require('glob');
-var path = require('path');
+const debug = require('debug')('objectify-folder');
+const fs = require('fs');
+const glob = require('glob');
+const path = require('path');
```

Comment:
Won't these break older versions of node? We should probably get a travis config in here.